src/main/java/org/venice/beachfront/bfapi/services/IABrokerService.java

```diff
@@ -100,7 +132,7 @@ public Scene getScene(String sceneId, String planetApiKey, boolean withTides) th
 					throw new RuntimeException(e);
```

Comment:
In all cases (here!) let's use our own exceptions. We need user-facing exceptions that are descriptive, and need a way to bubble this information up. 

Let's continue using UserException because it can be our generic way to represent errors/status codes